.github/workflows/ci.yml

```diff
@@ -44,22 +58,43 @@ jobs:
           large-packages: true
           docker-images: true
           swap-storage: true
-      - name: Install lld & GDAL & Protobuf
-        run: |
-          sudo apt-get update
-          sudo apt-get install lld libgdal-dev gdal-bin build-essential clang curl protobuf-compiler
-          sudo apt-get clean
-          export C_INCLUDE_PATH=/usr/include/gdal:$C_INCLUDE_PATH
-          export CPLUS_INCLUDE_PATH=/usr/include/gdal:$CPLUS_INCLUDE_PATH
-          sudo ldconfig
-      - name: Install Rustup
+      # TODO: Get Cache back
+      # - name: setup rust build cache
+      #   uses: Swatinem/rust-cache@v2
+      #   with:
+      #     # An explicit cache key that is used instead of the automatic `job`-based
+      #     # cache key and is thus stable across jobs.
+      #     # Default: empty
+      #     shared-key: ""
+
+      #     # An additional cache key that is added alongside the automatic `job`-based
+      #     # cache key and can be used to further differentiate jobs.
+      #     # Default: empty
+      #     key: ci_test_
+      - name: Testsuite
+        uses: ./.github/actions/coverage
+      - name: Adjust lcov.info
         run: |
-          curl --proto '=https' --tlsv1.2 --retry 10 --retry-connrefused -fsSL "https://sh.rustup.rs" | sh -s -- --profile minimal --default-toolchain none -y
-          echo "${CARGO_HOME:-$HOME/.cargo}/bin" >> $GITHUB_PATH
-      - name: Install SQLFluff
-        run: pip install sqlfluff==3.3.0
+          sed -i 's|/github/workspace/||g' lcov.info
```

Comment:
maybe add a comment and explain what you explained me earlier?

Reply:
[13d792f](https://github.com/geo-engine/geoengine/pull/1026/commits/13d792f10ed916224d8dc971f29ace29954c4114)